pkg/otbuiltin/builtin.go.h

```diff
@@ -46,7 +46,11 @@ _guint_to_pointer (guint u)
 }
 
 static void
+#ifdef GLIB_VERSION_2_58
+_g_clear_object (GObject **object_ptr)
```

Comment:
this function is not used at all, it can be safely removed.

I've worked on a patch for the same issue but haven't seen your PR, feel free to take a look at mine: https://github.com/giuseppe/ostree-go/tree/builtin-remove-unused-funcs

Reply:
@giuseppe Aha. I didn't know that these functions were not used at all.
I think it would be ok to remove them. 

You can ignore my PR. Feel free to create another PR.

Reply:
Opened a PR here: https://github.com/ostreedev/ostree-go/pull/25